src/main/java/uk/gov/hmcts/reform/sscs/utils/SscsCaseDataUtils.java

```diff
@@ -0,0 +1,149 @@
+import uk.gov.hmcts.reform.sscs.model.service.hearingvalues.CaseFlags;
+import uk.gov.hmcts.reform.sscs.model.service.hearingvalues.HearingWindow;
+import uk.gov.hmcts.reform.sscs.model.service.hearingvalues.HearingWindowDateRange;
+import uk.gov.hmcts.reform.sscs.model.service.hearingvalues.PartyDetails;
+
+import java.time.ZonedDateTime;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+public final class SscsCaseDataUtils {
+
+    private SscsCaseDataUtils() {
+        throw new IllegalStateException("Utility class");
+    }
+
+    public static String getHearingType(SscsCaseData caseData) {
+        return Optional.ofNullable(caseData.getAppeal())
+                .map(Appeal::getHearingType)
+                .orElse("");
+    }
+
+    public static List<String> getIssueCode(SscsCaseData caseData) {
+        if (Objects.nonNull(caseData.getIssueCode())) {
+            return List.of(caseData.getIssueCode());
+        }
+        return new ArrayList<>();
+    }
+
+    public static List<String> getFacilitiesRequired(SscsCaseData sscsCaseData) {
+        return Optional.ofNullable(sscsCaseData.getAppeal())
+                .map(Appeal::getHearingOptions)
+                .map(HearingOptions::getArrangements)
+                .orElse(new ArrayList<>());
+    }
+
+    public static String getCaseName(SscsCaseData sscsCaseData) {
+        return Optional.ofNullable(sscsCaseData.getAppeal())
+                .map(Appeal::getAppellant)
+                .map(Appellant::getName)
+                .map(Name::getFullName)
+                .orElse("");
+    }
+
+    public static int getNumberOfPhysicalAttendees(SscsCaseData sscsCaseData) {
+        int numberOfAttendees = 0;
+        // get a value if it is facetoface from hearingSubType -> wantsHearingTypeFaceToFace
+        if (Objects.nonNull(sscsCaseData.getAppeal())
+                && Objects.nonNull(sscsCaseData.getAppeal().getHearingSubtype())
+                && Objects.nonNull(sscsCaseData.getAppeal().getHearingSubtype().isWantsHearingTypeFaceToFace())
+                && sscsCaseData.getAppeal().getHearingSubtype().isWantsHearingTypeFaceToFace()) {
+            //appellants + dwp attendee (1) + judge (1) + panel members + representative (1)
+            numberOfAttendees = 1;
+            if (YesNo.isYes(sscsCaseData.getAppeal().getHearingOptions().getWantsToAttend())) {
+                numberOfAttendees++;
+            }
+
+            if (YesNo.isYes(sscsCaseData.getAppeal().getRep().getHasRepresentative())) {
+                numberOfAttendees++;
+            }
+            // TODO get it from SSCS-10243, when it is finished
+            numberOfAttendees += 0;
+
+            // TODO when panelMembers is created in caseData you will map it with the size of this value
+            //  (SSCS-10116)
+            numberOfAttendees += 0;
+        }
+        return numberOfAttendees;
+    }
```

Comment:
Can you move the body of this method to:
https://github.com/hmcts/sscs-hearings-api/blob/master/src/main/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsDetailsMapping.java#L147-L150

Reply:
Moved. Have changed return type from Number to int.